lib/doorkeeper/rake/db.rake

```diff
@@ -0,0 +1,51 @@
+# frozen_string_literal: true
+
+namespace :doorkeeper do
+  namespace :db do
+    desc 'Removes stale data from doorkeeper related database tables'
+    task cleanup: [
+                    'doorkeeper:db:cleanup:revoked_tokens',
+                    'doorkeeper:db:cleanup:expired_tokens',
```

Comment:
Let's indent it (all the array items) with just 2 spaces and final `]` under the `t` of `task`